### ISSUE TITLE
Remove Silverstripe Navigator

### ISSUE THREAD

**Issue by TheSceneman** [silverstripe/startup-theme#42]
Have been asked to remove this by James:
"It blocks the navigation bar, has an old look and feel and fails current accessibility standards."

**Comment by GuySartorelli** [comment]
We're not removing this. It was added intentionally.

> It blocks the navigation bar, has an old look and feel and fails current accessibility standards.

I would be open to PRs that stop it from blocking the nav bar, improve the look and feel, and improve accessibility of the feature, but not removing it.

If this is purely for demo, the template can be overridden in a demo project to remove it in that context.
